### PR TITLE
5 footer

### LINKED ISSUE #5: Footer
## Contents required
- contact information

### PR DESCRIPTION
**Pull request by Ellvius**
completed footer

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -1,7 +1,65 @@
...
+ * @author Lishin V S
+ */
+
+const Footer = ({
+    email= "ssladmin@nitc.ac.in",
+    phone= "0495 228 6864",
+    location= "ITL302 - 2nd Floor"
+}:{
+    email?: string,
+    phone?: string,
```
Comment (th3bossc) [inline src/components/Footer/index.tsx]: no need to take any props for footer, just make these static values.
We are not reusing the footer with different details right, so it doesn't make sense for it to be props

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -1,7 +1,65 @@
...
+        return () => window.removeEventListener("resize", handleResize);
+    },[]);
+
     return (
-        <footer>
+        <footer className="px-8 bg-[#202127] border-t border-white md:px-10">
+            <div className="font-bold text-sm flex flex-wrap justify-center  items-center py-5 sm:justify-between md:text-base">
+                {/* Contact Detail */}
+                <div className="flex flex-col mt-2 mb-8  lg:space-x-5  lg:mt-0  lg:mb-0 lg:flex-row">
+                    {[
```
Comment (th3bossc) [inline src/components/Footer/index.tsx]: move the array into a useMemo hook

**Review by th3bossc — CHANGES_REQUESTED** [review]
* remove the props and move it directy to the array 
* move array into a useMemo hook

**Review by th3bossc — APPROVED** [review]

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -1,7 +1,45 @@
...
+    const contactDetails = useMemo(() => [
+        { label: "email", value: "ssladmin@nitc.ac.in", icon: "/Footer/mail.svg" },
+        { label: "phone", value: "0495 228 6864", icon: "/Footer/phone.svg" },
+        { label: "location", value: "ITL302 - 2nd Floor", icon: "/Footer/location.svg" },
+    ], []);
+    
     return (
-        <footer>
+        <footer className="px-8 bg-[#202127] border-t border-white md:px-10">
+            <div className="font-bold text-sm flex flex-wrap justify-center  items-center py-5 sm:justify-between md:text-base">
```
Comment (NailaFathima) [inline src/components/Footer/index.tsx]: For screens >= lg, everything should be in 1 row, reduce font size if required
for all other screens, everything should be in 1 column
(including all contact details and copyrights)

**Review by NailaFathima — CHANGES_REQUESTED** [review]

File: src/components/Footer/index.tsx (hunk 3 above)
Comment (Ellvius) [reply src/components/Footer/index.tsx]: I have made the changes. Let me know if there is anything else to change

**Pull request merged**